src/main/java/org/madhuri/app/message/handler/StockHandler.java

```diff
@@ -0,0 +1,36 @@
+package org.madhuri.app.message.handler;
+
+import org.madhuri.app.message.gateway.StockGateway;
+import org.madhuri.app.message.model.Stock;
+
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+
+@Path("/stocks")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class StockHandler {
+	
+	private StockGateway stockGateway = new StockGateway();
```

Comment:
You should always call a service in handler, not a gateway.. so introduce a new stocks service layer and use it in handler